modules/cache/classes/Kohana/Cache/Memcached.php

```diff
@@ -0,0 +1,188 @@
+<?php defined('SYSPATH') or die('No direct script access.');
```

Comment:
we have removed `defined('SYSPATH') or die('No direct script access.');` from all files ;)

remove it and we are fine! :D